pygments/lexers/sql.py

```diff
@@ -595,63 +606,149 @@ class MySqlLexer(RegexLexer):
     tokens = {
         'root': [
             (r'\s+', Text),
-            (r'(#|--\s+).*\n?', Comment.Single),
-            (r'/\*', Comment.Multiline, 'multiline-comments'),
+
+            # Comments
+            (r'(?:#|--\s+).*(?=\n)?', Comment.Single),
+            (r'/\*\+', Comment.Special, 'optimizer-hints'),
+            (r'/\*', Comment.Multiline, 'multiline-comment'),
+
+            # Hexadecimal literals
+            (r"x'([0-9a-f]{2})+'", Number.Hex),  # MySQL requires paired hex characters in this form.
+            (r'0x[0-9a-f]+', Number.Hex),
+
+            # Binary literals
+            (r"b'[01]+'", Number.Bin),
+            (r'0b[01]+', Number.Bin),  # MySQL requires the 'b' to be lowercase.
```

Comment:
The case is not enforced (due to `flags = re.IGNORECASE`), but I don't think that matters...

Reply:
That's correct. I tried using syntax to temporarily disable `re.IGNORECASE` for the letter `b` (the syntax was something like `(?-i:b)` but this syntax is only available in Python 3.6 or higher. Hex, bin, and temporal literals all have this lowercase requirement but I forgot to remove this one comment. I'll get that fixed.

Note that I'll be using a history-rewriting force push to my branch.

Reply:
Okay, no history rewriting, GitHub appears to only update the PR if new commits are pushed.

Reply:
Hm, I never had problems with github picking it up when I force-pushed.